app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {

```

Comment:
http://stackoverflow.com/questions/14407884/how-can-i-secure-this-api-in-an-apk-file
http://stackoverflow.com/questions/14407884/how-can-i-secure-this-api-in-an-apk-file

Doesn't seem like there's really a great solution here. Setting up some server-side process that handles validating users and then using our API key might be the safest, not sure how much we want to put into this